core/src/main/java/org/apache/calcite/rex/RexCallBinding.java

```diff
@@ -122,6 +122,10 @@ public static RexCallBinding create(RelDataTypeFactory typeFactory,
     return RexUtil.isLiteral(operands.get(ordinal), allowCast);
   }
 
+  public List<RexNode> getOperands() {
+    return operands;
+  }
```

Comment:
Follow the `SqlCallBinding` name pattern, use `operands` or `operand`.

Reply:
Done